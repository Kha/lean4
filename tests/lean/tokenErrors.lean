#check 'hi
example : Nat := 0  -- recover

#check '\y'
example : Nat := 0  -- recover

/-
-- these are all "fatal" 
#check "hi
#check hi.Â«
-/

#print Nat
/-
