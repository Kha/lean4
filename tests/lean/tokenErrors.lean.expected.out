tokenErrors.lean:1:10: error: missing end of character literal
tokenErrors.lean:4:9: error: invalid escape sequence
tokenErrors.lean:15:0: error: end of input
